### ISSUE TITLE
Can't get sms or call code

### ISSUE THREAD

**Issue by M4STERANGEL** [joaoricardo000/whatsapp-bot-seed#66]
INFO:yowsup.common.http.warequest:{"status":"fail","reason":"old_version"}

status: fail
reason: old_version

This error happens me every time that I enter my phone to get the key

**Comment by x23piracy** [comment]
Try to update yowsup to latest version, if this is not working register with an android device and use the following app to read the credentials:

https://github.com/mgp25/Chat-API/wiki/Extracting-password-from-device
https://www.mgp25.com/downloads/pw.apk

[https://camo.githubusercontent.com/f4c4c9f51e4a30bbcb1f05613a07fbe73bb901aa/687474703a2f2f636c2e6c792f696d6167652f30323073305930563269315a2f534332303134303932352d3030353035352e706e67](url)

**Comment by M4STERANGEL** [comment]
Okey, I have the key, now?
Because is a bit difficult to understand setup.py

**Comment by K4CZP3R** [comment]
Change env to Android

**Comment by K4CZP3R** [comment]
https://github.com/tgalal/yowsup/pull/1336/files

**Comment by x23piracy** [comment]
Is this because they stopped supporting symbian devices?

**Comment by K4CZP3R** [comment]
Yup, but only for register. You will need symbian env (not sure for 100%) to recieve whatsapp group msg

**Comment by x23piracy** [comment]
Ok, means doing the change temp. to reg. then change to sym again.

Thanks

Regards X23

**Comment by M4STERANGEL** [comment]
I can't sign in, the same problem
@K4CZP3R
